tests/unittest/llmapi/test_kv_cache_dtype_override.py

```
@@ -57,15 +57,34 @@ def test_kv_cache_config_dtype_validation():
 
+    cfg = KvCacheConfig(dtype="FP8_DS_MLA")
+    assert cfg.dtype == "fp8_ds_mla"
+
     with pytest.raises(ValueError, match="kv_cache_config.dtype must be one of"):
         KvCacheConfig(dtype="invalid_dtype")
 
 
+def test_kv_cache_config_rejects_fp8_ds_mla_pool_rebalance():
+    with pytest.raises(
+        ValueError,
+        match="fp8_ds_mla.*enable_kv_pool_rebalance",
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)